lib/internal/fs/rimraf.js

```diff
@@ -0,0 +1,249 @@
+// The code in this file is adapted from the rimraf module.
```

Comment:
Should we include a link to where the source lives?

@isaacs do you have a preference?